### ISSUE TITLE
Is searching from inside the component possible?

### ISSUE THREAD

**Issue by a-singer** [TheQwertiest/foo_spotify#52]
Hi,

I'm sorry for the foolish question but I'm not sure what's going on with search. On the one hand, I can find help on putting in queries. On the other, I can't seem to find a field to put them in. Again, I see that the todo list seems to list improvements to the search. I should say that I'm blind and therefore the search field, if it exists, may not be easily visible to the screen reader I'm using. Is there such a field, or do I have to find URIs on the Spotify site/player and add them after having copied them from the site? This last is what I'm doing now, but it would be nice to search inside Foobar itself

Thanks..

**Comment by TheQwertiest** [comment]
> Is there such a field, or do I have to find URIs on the Spotify site/player and add them after having copied them from the site? 

Yes, currently you have to copy-paste URI or URLs from Spotify directly (as mentioned in https://theqwertiest.github.io/foo_spotify/docs/faq/#how-to-actually-play-music-from-spotify).

But yes, as you've noticed, an integrated Spotify search engine is on my to-do list.
